dist/cookie/{{cookiecutter.project_slug}}/package.json

```diff
@@ -33,7 +33,8 @@
     "react-native-vector-icons": "^9.2.0",
     "react-native-web": "^0.18.10",
     "react-redux": "^8.0.5",
-    "redux": "^4.2.1"
+    "redux": "^4.2.1",
+    "react-native-paper": "^4.*"
```

Comment:
Can you be a bit more specific with the version? 
I guess a better question is: was there any issues testing the latest react native paper with the current scaffold?
I remember there was an issue with the previous scaffold version, that's why the SPIKE investigated using this one, but there was a comment there saying that we should likely try to get it working with the latest, if possible. Basically just reinstall in your demo app the latest Paper version and check how it runs

If there are some errors, let me know

Reply:
Done.